compiler/rustc_codegen_cranelift/example/mini_core.rs

```diff
@@ -517,7 +517,7 @@ impl<T: ?Sized + Unsize<U>, U: ?Sized> CoerceUnsized<Box<U>> for Box<T> {}
 
 impl<T: ?Sized> Drop for Box<T> {
     fn drop(&mut self) {
-        // drop is currently performed by compiler.
+        libc::free(self.0.pointer.0 as *mut u8);
```

Comment:
This should keep a comment that dropping the inner value is still done by the compiler.